.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 88
```

Comment:
Any specific reason for not including, flake8 to pre-commit config?

Reply:
Black doesn't always agree with flake8. For example E203.